thinc/loss.py

```diff
@@ -342,6 +351,22 @@ def _make_mask(missing, guesses) -> Floats2d:
     return mask
 
 
+def _make_xp_mask(guesses, truths, missing_value) -> Floats2d:
```

Comment:
This looks good to me in general, but it annoys me slightly that the signature of `_make_xp_mask` is so different to `_make_mask` in the order of the arguments. Maybe `_make_mask` should have had the order of arguments reversed, and it is a private function so maybe it wouldn't even hurt to change this on the fly, just to have some kind of consistency between the function declarations?

Reply:
We should probably also reconsider the naming of these two functions...

Reply:
I changed the order of `_make_xp_mask` to correspond to `convert_truths`.

Do you have suggestions for the name? Maybe `_missing_value_mask`?

Reply:
`_make_mask_by_value`?

Reply:
👍🏻 Done!

Reply:
Oh hm wait, I'm confused, I thought we'd have `_make_mask` as before and `_make_mask_by_value` as the latter checks on the missing value?

Reply:
Makes sense! I'll update the names.

Reply:
I did do a force push this time, to avoid a commit which fixes a commit ;).